tools/Makefile

```diff
@@ -33,8 +32,6 @@ SET_VERSION_SCRIPTS = \
 	bin/gpexpand \
```

Comment:
Why leave gpexpand?  This has been replaced with the HAWQ 2.0 mechanisms around adding/removing nodes.